snapcraft/tests/test_yaml.py

```diff
@@ -283,7 +283,89 @@ def test_invalid_yaml_invalid_name_as_number(self, mock_loadPlugin):
             snapcraft.yaml.Config()
 
         self.assertEqual(raised.exception.message,
-                         '1 is not of type \'string\'')
+                         'The \'name\' property does not match the required '
+                         'schema: 1 is not of type \'string\'')
```

Comment:
If you use double quotes then you won't have to escape the single quotes. Same for the rest of the tests.